packages/x-data-grid/src/models/gridExport.ts

```diff
@@ -40,6 +40,12 @@ export interface GridFileExportOptions<Api extends GridApiCommon = GridApiCommun
    * @returns {GridRowId[]} The list of row ids to export.
    */
   getRowsToExport?: (params: GridGetRowsToExportParams<Api>) => GridRowId[];
+  /**
+   * If `true`, the formulas in the cells will be escaped.
+   * It is recommended to set this to `true` for security reasons.
+   * @default false
```

Comment:
@mui/xgrid What do you think about making it `true` by default?
Technically, it's a breaking change, but it's also a security fix.
Any objections?

Reply:
No objection, we should change it.